docs/source/en/_toctree.yml

```diff
@@ -1,144 +1,158 @@
+  sections:
+  - title: Trainer API
+    isExpanded: false
     sections:
-    - local: tasks/image_captioning
-      title: Image captioning
-    - local: tasks/document_question_answering
-      title: Document Question Answering
-    - local: tasks/visual_question_answering
-      title: Visual Question Answering
-    - local: tasks/text-to-speech
-      title: Text to speech
-    title: Multimodal
-  - isExpanded: false
+    - local: training
+      title: Fine-tune a pretrained model
+    - local: trainer
+      title: Trainer
+    - local: hpo_train
+      title: Hyperparameter Search using Trainer API
+    - local: run_scripts
+      title: Train with a script
+  - title: Distributed training
+    isExpanded: false
     sections:
-    - local: generation_strategies
-      title: Customize the generation strategy
-    title: Generation
-  - isExpanded: false
+    - local: accelerate
+      title: Set up distributed training with 🤗 Accelerate
+    - local: perf_train_gpu_many
+      title: Multiple GPUs and parallelism
+    - local: fsdp
+      title: Fully Sharded Data Parallel
+    - local: deepspeed
+      title: DeepSpeed
+    - local: perf_train_cpu_many
+      title: Distributed CPU training
+  - title: Hardware-specific training
+    isExpanded: false
+    sections:
+    - local: perf_train_gpu_one
+      title: Methods and tools for efficient training on a single GPU
+    - local: perf_train_cpu
+      title: Efficient training on CPU
+    - local: perf_train_tpu_tf
+      title: Training on TPU with TensorFlow
+    - local: perf_train_special
+      title: PyTorch training on Apple silicon
+    - local: perf_hardware
+      title: Custom hardware for training
+  - local: peft
+    title: Load and train adapters with 🤗 PEFT
+  - local: sagemaker
+    title: Run training on Amazon SageMaker
+  - local: debugging
+    title: Debugging
+  - local: model_memory_anatomy
+    title: Model training anatomy
+- title: Inference
+  sections:
+  - title: Pipeline API
+    isExpanded: false
+    sections:
+    - local: pipeline_tutorial
+      title: Run inference with pipelines
+    - local: pipeline_webserver
+      title: Pipelines for webserver inference
+    - local: add_new_pipeline
+      title: How to add a pipeline to 🤗 Transformers?
+  - title: LLMs
+    isExpanded: false
     sections:
-    - local: tasks/idefics
-      title: Image tasks with IDEFICS
     - local: tasks/prompting
       title: LLM prompting guide
-    title: Prompting
-  title: Task Guides
-- sections:
-  - local: fast_tokenizers
-    title: Use fast tokenizers from 🤗 Tokenizers
+    - local: llm_optims
+      title: LLM inference optimization
+    - local: llm_tutorial
+      title: Generation with LLMs
+    - local: generation_strategies
+      title: Customize the generation strategy
+    - local: llm_tutorial_optimization
+      title: Getting the most out of LLMs
+    - local: perplexity
+      title: Perplexity of fixed-length models
+  - title: Chat models
```

Comment:
perhaps move this down a level (i.e. inside LLMs)? a Chat model is a subtype of LLM